dev/content/pages/About.md

```diff
@@ -105,15 +105,18 @@ save_as: index.html
 <!-- Left -->
 <div class="about-box">
 <h1 style="margin-top:0; text-align: left;">About Me</h1>
-    <p>I am a data scientist at Google where I use data to inform decisions, although all my opinions here are my own. I have previously worked at Sweetgreen, designing systems that would benefit team members and communities through sustainable and healthy food, and SpaceX, creating tools that would ultimately launch rocket ships. I feel fortunate to have worked with amazing teams throughout my career.</p>
-    <p>There are 3 things I strive to be:</p>
+    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously worked at Sweetgreen and SpaceX, doing the same, using data to inform staffing strategy and rocket launch cadence.</p>
+    
+    <p>To become a better "decision" scientist I'm currently focusing on three topics</p>
     <ol>
-        <li>An expert computational mathematician</li>
-        <li>An effective community member and leader</li>
-        <li>A knowledge contributor</li>
+        <li>Organizational Influence and Leadership</li>
+        <li>Advanced applied mathematics e.g. Causal Inference, Decistion Theory</li>
+        <li>Business Domain expertise</li>
     </ol>
-    <p>I always thought it was cool that people would come together and use math to build awesome things, whether it be rocket ships or computational statistics libraries. One of my most rewarding experiences came from contributing to the open source community. Since then, I have been a core contributor to <a href="https://arviz-devs.github.io/arviz/index.html">ArviZ</a> and <a href="https://docs.pymc.io/">PyMC3</a>.</p>
-    <p>I believe in the possibility of creating great outcomes with math. The variety and volume of data has increased faster than the knowledge of how to make full use of it, and I am working to fill the gap. You'll see my public efforts in my open source profile, YouTube channel, book on Bayesian statistics, online course on applied Bayesian statistics, and many more.</p>
+    <p>I like to share and teach the skills I've found useful from my career, whether it be here or in my blog, on YouTube, or as part of my course series</p>
+    <p>If you'd prefer to updates pushed to you can subscribe to my mailing list </p>
```

Comment:
1. ):<
2. Can you read what you originally wrote? 😂 
3. Here are some options:

Minimal corrections to what you wrote:
```suggestion
    <p>If you'd prefer to have updates pushed to you, you can subscribe to my mailing list.</p>
```
What I prefer:
```suggestion
    <p>If you prefer to get notified whenever I send updates, consider subscribing to my mailing list.</p>
```